tests/sentry/api/endpoints/test_organization_member_index.py

```diff
@@ -143,7 +143,7 @@ def test_existing_user_for_invite(self):
                 self.url, {"email": user.email, "role": "member", "teams": [self.team.slug]}
             )
 
-        assert resp.status_code == 409
+        assert resp.status_code == 400
```

Comment:
Lost a bit of status code information by using the serializer. Not too concerned. Though I suppose it breaks us as an external API.